### PR TITLE
Adding Back Fill Hasal Result feature into Trigger Server base on Builds

### PR DESCRIPTION
**Pull request by askeing**

### CODE REVIEW

**Comment by askeing** [comment]
@ShakoHo @ypwalter r?

File: ejenti/server_jobs/backfill_trigger.py [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+        StatusFileCreator.create_status_file(job_id_fp, StatusFileCreator.STATUS_TAG_PULSE_TRIGGER_BACKFILL, 100)
+
+        # check queue
+        queue_exists = HasalPulsePublisher.check_pulse_queue_exists(username=self.pulse_username,
+                                                                    password=self.pulse_password,
+                                                                    topic=topic)
+
+        if not queue_exists:
+            logging.error('There is not Queue for Topic [{topic}]. Message might be ignored.'.format(topic=topic))
+
```
Comment (ShakoHo) [inline ejenti/server_jobs/backfill_trigger.py:60]: if the queue does not exist, do we need to continue the following steps?

**Review by ShakoHo — APPROVED** [review]

File: ejenti/server_jobs/backfill_trigger.py (hunk 1 above)
Comment (askeing) [reply ejenti/server_jobs/backfill_trigger.py:60]: It's a trade-off between cpu resource and case running.
Not only publisher can create queue, any one of consumers (agents) also can create queue.
It still has chance to submit tasks after queue be created by consumer.

File: ejenti/server_jobs/backfill_trigger.py [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+from lib.modules.build_information import BuildInformation
+from lib.common.commonUtil import CommonUtil
+from ejenti.pulse_modules.hasalPulsePublisher import HasalPulsePublisher
+
+
+class BackFillTrigger(object):
+
+    PLATFORM = 'win64'
+    RESULT_AMOUNT = 6
+    WAIT_TIMEOUT_DELTA_HOUR = 8
```
Comment (askeing) [inline ejenti/server_jobs/backfill_trigger.py]: it seems like I should move this parameter out to config file

**Pull request merged**